jormungandr/src/main.rs

```diff
@@ -170,9 +170,17 @@ fn start_services(bootstrapped_node: BootstrappedNode) -> Result<(), start_up::E
             transaction_box: fragment_msgbox,
             block_box: block_msgbox,
         };
+        let block0 = bootstrapped_node
+            .blockchain
+            .lock_read()
+            .tip
+            .ledger()
```

Comment:
I realize it was not made clear in the original issue, but I think this is was not the intended source of information. I think we want to list the stake pool at the moment it has been snapshotted for the current leadership, not the current stake pool.

that being said, we can fix it later and accept this as a putting most of the REST call in place. Or we can also offer both the current tip and the snapshot (possibly with a different rest endpoint)